src/Finder/ModuleFileFinder.php

```
@@ -46,4 +57,45 @@ protected function _filter(SplFileInfo $fileInfo)
     {
         return $fileInfo->isReadable();
```

Perhaps, it is better to let the locator [throw](https://github.com/RebelCodecom/module-locator/blob/develop/src/FileLocator.php#L149) with helpful info, instead of just silently ignoring the file to leave the user wondering why their config is not having an effect?

---

That's true. I'll make the change.

---

@mecha, you seem to have completely misunderstood me. Please read my comment again, and follow the link.
Also, it has been recently discussed whether the locator should be locating modules from a config instead of files. In this case `ModuleFileFinder` will be supplying configuration to a config repository, instead of a locator. Nevertheless, the plan is the same: finder should only find the files; whatever reads the files should attempt to read a configuration file, and then decide on the course of action. In the event that the file is not readable, *that component* may log this, or most probably throw an exception of its own - with helpful info attached in it, as I mentioned in the comment, and ideally using an exception factory method.